dnf-behave-tests/fixtures/specs/createrepo_c-ci-notinstallable/createrepo_c-0.14.4.spec

```diff
@@ -0,0 +1,145 @@
+%global libmodulemd_version 2.3.0
+
+Summary:        Creates a common metadata repository
+Name:           createrepo_c
```

Comment:
I find it confusing you are using the `createrepo_c` package as test data for `createrepo_c` tests. It appears like this package here somehow could have a specific role in the tests, until you dig a bit and find out it can just be arbitrary test data. Also it will be unnecessarily popping up while grepping for e.g. "createrepo_c".

And, it seems this is the full spec file, which contains a lot of (I assume) redundant things, like listing all the dependencies and installs a lot of files, of which you probably only need a few?

Seems you don't need too many packages for the tests, which is good, why not further reduce the footprint and only leave what's actually important in the test data?